elements/sdk/eos-sdk.bst

```diff
@@ -0,0 +1,26 @@
+kind: autotools
+
+build-depends:
+- freedesktop-sdk.bst:components/git.bst
+- freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
+- gnome-sdk.bst:sdk/gtk-doc.bst
+
+depends:
+- gnome-sdk-sdk/gtk+-3.bst
+- gnome-sdk.bst:sdk/glib.bst
+- gnome-sdk.bst:sdk/gobject-introspection.bst
+- gnome-sdk.bst:sdk/json-glib.bst
+- sdk/eos-metrics.bst
+
+variables:
+  conf-local: |
+    --disable-gtk-doc \
+    --disable-gir-doc \
+    --disable-js-doc \
+    --disable-webhelper
+
+sources:
+- kind: git_tag
+  url: github_com:endlessm/eos-sdk.git
+  track: sdk-6
+  ref: Release_3.3.18-56-g046c835870f14d53297435dc022740185c756cd1
```

Comment:
This is a bit surprising…

On one hand you're saying you want to track the branch `sdk-6` but to only take tags from it (using the `git_tag` source).

On the other hand you explicitly take a commit which isn't on a tag (it's 56 commits after the `Release_3.3.18` tag).

Was that expected?

Reply:
Yeah, git_tag by default tracks the latest commit - it only tracks the latest tag if you specify `track_tags: True`. (Which is kind of backwards, now that you mention it). This is the case for a lot of our dependencies, although it would be nice to track tags for repos that we know are good about tagging releases.

Reply:
Ah, I see where that's weird. In gnome-build-meta, they stick with the default track-tags: False. In freedesktop-sdk, the default is track-tags: True. This one is closer to gnome-build-meta, albeit for no particular reason :)